packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js

```diff
@@ -26,6 +26,9 @@ export default class ProductSearchResults extends LightningElement {
     };
     sortRuleValue = '';
 
+    headertextprefix = '';
+    headerText = '';
```

Comment:
Every where else this variable isn't camel cased

Reply:
Made the correction. Thx!